### PR TITLE
Item check 10250 10293

### PR DESCRIPTION
**Pull request by helixhamin**
Okay, should be good for pull now.

### CODE REVIEW

File: scripts/globals/items/chocobo_shirt.lua [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+elseif (dayElement == 3)
+    target:addItem(4098,math.random(4,12)); then
+elseif (dayElement == 4)
+    target:addItem(4097,math.random(4,12)); then
+elseif (dayElement == 5)
+    target:addItem(4100,math.random(4,12)); then
+elseif (dayElement == 6)
+    target:addItem(4102,math.random(4,12)); then
+else (dayElement == 7)
+    target:addItem(4103,math.random(4,12)); then
```
Comment (TeoTwawki) [inline scripts/globals/items/chocobo_shirt.lua]: Newp! broken syntax, bad formatting, and there's a better way to do this:
Welcome to the wonderful world of tables and arrays N stuffs!
```lua
-----------------------------------------
-- ID =  10293
-- Chocobo Shirt
-- Dispense =  Crystals
-----------------------------------------


-----------------------------------------
-- OnItemCheck
-----------------------------------------

function onItemCheck(target)
    local result = 0;
    if (target = getFreeSlotsCount() == 0) then
        result = 308;
    end
    return result;
end;

-----------------------------------------
-- OnItemUse
-----------------------------------------

function onItemUse(target)
    local day = VanadielDayElement()+1;
    local crystal =
    {
        [1] = 4096, -- Fire
        [2] = 4099, -- Earth
        [3] = 4101, -- Water
        [4] = 4098, -- Wind
        [5] = 4097, -- Ice
        [6] = 4100, -- Thunder
        [7] = 4102, -- Light
        [8] = 4103  -- Dark
    };
    print(crystal[day]);
    target:addItem(crystal[day],math.random(2,12));
end;
```

Comment (TeoTwawki) [inline scripts/globals/items/chocobo_shirt.lua]: I left a print in, you'll want to test with that and then remove it.

Comment (helixhamin) [inline scripts/globals/items/chocobo_shirt.lua]: Thanks for the advice. It looks much better now. But, one question, why the VanadielDayElement()**+1**? Also, the array should start at 0 for fireday, correct? I tested my if statement, and it worked, so the material I found was correct. Fireday = day 0, ++.

Comment (TeoTwawki) [inline scripts/globals/items/chocobo_shirt.lua]: I rightly should have done
```lua
    local crystal =
    {
        4096, -- Fire
        4099, -- Earth
        4101, -- Water
        4098, -- Wind
        4097, -- Ice
        4100, -- Thunder
        4102, -- Light
        4103  -- Dark
    };
```
and then explained how lua does all its indexes starting at 1 normally, or else forced a zero index with
```lua
    local crystal =
    {
        [0] = 4096, -- Fire
```
somehow I half did both. I'll blame my lack of caffeine. Sounds legit!

File: scripts/globals/items/chocobo_shirt.lua [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        4096, -- Fire
+        4099, -- Earth
+        4101, -- Water
+        4098, -- Wind
+        4097, -- Ice
+        4100, -- Thunder
+        4102, -- Light
+        4103  -- Dark
+    };
+    print(crystal[day]);
```
Comment (takhlaq) [inline scripts/globals/items/chocobo_shirt.lua]: sneaky debug print!

Comment (helixhamin) [inline scripts/globals/items/chocobo_shirt.lua]: Yeah, leaving it until I can test. I will remove tomorrow if a test tonight goes okay.

**Comment by helixhamin** [comment]
@takhlaq Okay, print statement commented out, ready as long as no issues are flagged in travis/appveyor

**Pull request merged**